src/ert/scheduler/lsf_driver.py

```diff
@@ -266,6 +266,7 @@ async def submit(
             retry_interval=self._sleep_time_between_cmd_retries,
         )
         if not process_success:
+            self._job_error_message[iens] = process_message
```

Comment:
Can a job have multiple errors? 
Maybe we should do `self._job_error_message[iens] += process_message` instead?

Reply:
Maybe, can include this one.

Reply:
Actually, changed my mind - we do care only about the last error msg, I think.